docs/en-US/component/divider.md

```diff
@@ -37,7 +47,8 @@ divider/vertical-divider
 
 ## Divider Attributes
 
-| Attribute        | Description                               | Type   | Accepted Values       | Default    |
-| ---------------- | ----------------------------------------- | ------ | --------------------- | ---------- |
-| direction        | Set divider's direction                   | string | horizontal / vertical | horizontal |
-| content-position | customize the content on the divider line | String | left / right / center | center     |
+| Attribute        | Description                               | Type   | Accepted Values                                                    | Default    |
+| ---------------- | ----------------------------------------- | ------ | ------------------------------------------------------------------ | ---------- |
+| direction        | Set divider's direction                   | string | horizontal / vertical                                              | horizontal |
+| border-style     | Set the style of divider                  | string | dotted / dashed / solid / double / groove / ridge / inset / outset | solid      |
```

Comment:
It's ok. Just a suggestion. Maybe we can leave a link directly instead of writing so many values.

https://developer.mozilla.org/zh-CN/docs/Web/CSS/border-style

Reply:
ok，

Reply:
Allow me to nitpick, because it's an English document by default. I think we don't need to write "请查看".

Reply:
ok,